packages/client/components/AddTaskButton.tsx

```diff
@@ -22,7 +22,7 @@ const AddTaskIcon = styled(Icon)({
   fontSize: 20,
   width: 20,
   height: 20,
-  margin: '0 4px'
+  margin: '0 4px 0 0'
```

Comment:
+1 can you fix it on the `AddPollButton` https://github.com/ParabolInc/parabol/blob/d82610163cfb75ed13f23d6b1bdb75748832e176/packages/client/components/AddPollButton.tsx#L25 as well? This way we don't forget it.

Reply:
Will do, thank you! 🙌